satpy/readers/mviri_l1b_fiduceo_nc.py

```diff
@@ -153,19 +153,15 @@
 """
 
 import abc
-from functools import lru_cache
 import warnings
+from functools import lru_cache
 
 import dask.array as da
 import numpy as np
 import xarray as xr
 
 from satpy import CHUNK_SIZE
-from satpy.readers._geos_area import (
-    sampling_to_lfac_cfac,
-    get_area_definition,
-    get_area_extent
-)
+from satpy.readers._geos_area import get_area_definition, get_area_extent, sampling_to_lfac_cfac
```

Comment:
Why were those changed to be on one line?  Elsewhere the change was the other way around.

Reply:
Maybe they fit within 120 characters?